packages/sdk/src/apiClient/apiClient.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import axios, { AxiosInstance } from "axios";
+import { AuthProvider } from "./authProvider";
+
+/**
+ * Initializes new Axios instance with specific auth provider
+ *
+ * @param apiEndpoint - Base url of the API
+ * @param authProvider - Auth provider that injects authentication info to each request
+ * @returns axios instance configured with specfic auth provider
+ *
+ * @example
+ * ```typescript
+ * const client = createApiClient("https://my-api-endpoint-base-url", new BasicAuthProvider("xxx","xxx"));
```

Comment:
Maybe we can use `BearerTokenAuthProvider` here as example because we don't provide BaiscAuthProvider now.

Reply:
Will soon add the BasicAuthProvider. Thx